auth/google.go

```diff
@@ -123,3 +124,17 @@ func GoogleTokenSource() (oauth2.TokenSource, error) {
 	}
 	return conf.TokenSource(oauth2.NoContext, tok), nil
 }
+
+// Fetch from Google Compute Engine's metadata server to retrieve
+// an access token for the provided account.
+func GoogleServiceClient() *http.Client {
+	return &http.Client{
+		Transport: &oauth2.Transport{
+			Source: google.ComputeTokenSource(""),
```

Comment:
Empty string means it will use the "default" service account which seems to work fine. Should I expose that to the API?
https://godoc.org/golang.org/x/oauth2/google#ComputeTokenSource